crates/fiber-lib/src/fiber/channel.rs

```diff
@@ -2941,6 +2945,15 @@ impl TlcStatus {
     }
 }
 
+bitflags! {
+    #[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+    #[serde(transparent)]
+    pub struct AppliedFlags: u8 {
+        const APPLIED_ADD = 1;
```

Comment:
We can remove the duplicated prefix `APPLIED_`

Reply:
good idea.